Makefile.kokkos

```diff
@@ -510,6 +545,21 @@ ifeq ($(KOKKOS_INTERNAL_USE_CUDA), 1)
   tmp := $(call kokkos_append_header,"$H""define KOKKOS_COMPILER_CUDA_VERSION $(KOKKOS_INTERNAL_COMPILER_NVCC_VERSION)")
 endif
 
+ifeq ($(KOKKOS_INTERNAL_USE_OPENACC), 1)
+  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_OPENACC")
+ifeq ($(KOKKOS_ENABLE_CUDA_ATOMIC_INTRINSICS), 1)
+  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_CUDA_ATOMIC_INTRINSICS")
+endif
+ifeq ($(KOKKOS_ENABLE_COLLAPSE_HIERARCHICAL_CONSTRUCTS), 1)
+  tmp := $(call kokkos_append_header,"$H""define KOKKOS_ENABLE_COLLAPSE_HIERARCHICAL_CONSTRUCTS")
+endif
+endif
+
+ifeq ($(KOKKOS_ENABLE_OPENACC), 1)
+  a = $(shell echo OPENACC enabled)
+  $(info $(a))
+endif
+
```

Comment:
```suggestion
```
I don't think that we print here for any other backends.

Reply:
Done!